Misc/NEWS.d/next/Library/2020-05-19-23-16-53.bpo-39673.ppfScY.rst

```diff
@@ -0,0 +1 @@
+Map ``errno==ETIME`` to :exc:`TimeoutError`.
```

Comment:
Could you include in here that it's an OSError whose behavior is being changed? It's relevant that it's an OSError with errno==ETIME that's becoming a TimeoutError. Maybe just add "OSError with" after "Map"?

Reply:
Since it's trivial enough, I made this edit myself.